content/en/monitors/types/cloud_cost.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cloud Cost Monitor
+kind: documentation
+description: "Monitor costs associated with cloud platforms."
+aliases :
+    - /monitors/monitor_types/custom_check
+    - /monitors/create/types/custom_check/
+further_reading:
+- link: "https://docs.datadoghq.com/cloud_cost_management/?tab=aws#overview"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/monitors/notify/"
+  tag: "Documentation"
+  text: "Configure your monitor notifications"
+- link: "/monitors/notify/downtimes/"
+  tag: "Documentation"
+  text: "Schedule a downtime to mute a monitor"
+- link: "/monitors/manage/status/"
+  tag: "Documentation"
+  text: "Consult your monitor status"
+---
+
+## Overview
+Get proactive monitor notifications on cost changes and mitigate unnecessary spending. Cloud Cost Monitors help you to identify cost changes as early as possible so you can understand cost changes and communicate this with engineers and finance teams. Configure your alerts to catch unexpected cost changes. 
```

Comment:
Potential rephrase: "Get proactive notifications on cost changes to help mitigate unexpected cloud spend. Cloud Cost Monitors help you identify cost changes quickly so you can investigate why costs changed. Configure your alerts to catch unexpected cost changes."